docs/docs/cli/integrations.md

```diff
@@ -0,0 +1,42 @@
+
+## Overview
+
+Mailscript provides integrations with the external services Google Drive and Zoom. These integrations provide validated OAuth tokens to Mailscript Actions after the integrations are setup. The provided OAuth tokens can be used to interact directly with the integrated service.
+
+## Google Drive
+
+The Google Drive Integration provides an OAuth token with the permissions defined by [drive.files](https://www.googleapis.com/auth/drive.file). This allows for viewing and managing Google Drive files and folders that you have opened or created with this app.
+
+### Setup
+
+To setup the Google Drive Integration, simply ensure you are logged in under the CLI with:
```

Comment:
To setup the Google Drive Integration, ~simply~ **first**  ensure you are logged in under the CLI with